rules-topics/16_life-and-death.md

```diff
@@ -88,3 +88,28 @@ Killing Blow is a 3-count that can be interrupted. When Killing Blowed, the targ
 A character whose body has been dismembered while **Bleeding Out** or **Dead** may still be healed or revived. The part of the character that can be so affected is represented by and located wherever the player of that character is. The spirit-containing body of a character may be carried by another character, however, the person carrying the body may not run or move faster than the player of the character being carried.
 
 Dismembering a character for the purpose of attempting to use a smaller body part to heal, revive, or even hide the character is not any different than if the character were not dismembered. If a character is **Bleeding Out** or **Dead** and someone wishes to bring that character to a healer, or move the body away from where it fell, they may role-play carrying the body by picking it up on a three count _(ex: "One I pick you up… Two I pick you up… Three I pick you up…")_ and carrying it elsewhere. The carrying player may not run, and both players must roleplay maintaining contact and remain within arm's reach of each other. If any effect would forcibly move the carrier away at a pace faster than a walk, or would otherwise render them unable to hold game items in their hands, the character must drop the body on a One Count _(ex: "One I drop you")_ and may only resume carrying them on a new three count.
+
+## Metabolisms
+Most people and creatures have a regular metabolism. Characters with "no metabolism" or "undead metabolism" are affected by some game effects differently.
+
+### No Metabolism
+- Crumble at 0 Body
+- Immune to Toxin
+- Immune to Berserk, Command: Confuse, Death, Boon of Nature, Drain, First Aid, Heal Wounds, Hearthstone, Infect, Life, Regenerate, Revive, Revivify, Sleep, Stun, Vampiric Touch, Waylay
+
+> _Generally, a creature that has no metabolism cannot be healed by typical means, but will have an alternate method of being healed. For instance, a Fire Elemental would be immune to standard healing effects, but could be healed by Flame._ 
+
+
+### Undead Metabolism
+Same effects as per "No Metabolism", except the character is affected normally by Heal Wounds. They are also affected by effects and abilities that specifically target Undead (such as Harm Undead).
```

Comment:
Please clarify "same effects" here?  Does this include the "Immune to Berserk ... [ ] Waylay"?  If yes, same clarification request about "receiving" versus self-use.

Reply:
I don't understand, it's the same list of effects as "No Metabolism"

It seemed silly to explicitly duplicate the list of effects.